Phases/README.md

```diff
@@ -0,0 +1,37 @@
+# Phases
+
+With the changes introduced in "version 1.5" of the guild, there are now 5 sequential phases that learners move through.
+
+1. [Foundations 1](#foundations-1)
+1. [Foundations 2](#foundations-2)
+1. [Integrated Practice Curriculum (IPC)](#integrated-practice-curriculum-ipc)
+1. [Apprenticeship](#apprenticeship)
+1. [Career Prep](#career-prep)
+
+To move from phases 1->2, 2->3, and 3->4, learners must pass an assessment. See more details in the [Assessments](./Assessments) section.
+
+## Foundations 1
+
+Develop fundamental skills so that you can begin to take on more advanced topics and projects.
+
+## Foundations 2
+
+Build upon the development foundations from phase 1 while moving into the world of web development.
+
+To start Phase 2, you must pass the [Phase 1 assessment](./Assessments/Phase_1.md).
+
+## Integrated Practice Curriculum (IPC)
```

Comment:
IPC also stands for Inter Process Communication... potentially confusing for geeks.  😄